digiroad2-api-oth/src/test/scala/fi/liikennevirasto/digiroad2/IntegrationApiSpec.scala

```diff
@@ -274,4 +278,27 @@ class IntegrationApiSpec extends FunSuite with ScalatraSuite with BeforeAndAfter
     returntxt should be ("LINESTRING ZM (0.0 0.0 0.0 0.0, 1.0 0.0 0.5 1.0, 4.0 4.0 1.5 6.0)")
 
   }
+
+  test("sevenRestrictionToApi length_limits"){
```

Comment:
We can put a better name on this like for example:
"Validate if sevenRestriction JSON generator return all required keys"

Reply:
Because this sevenRestrictionToApi it's used not only at length_limits

Reply:
Applied as suggested